package.json

```diff
@@ -36,5 +38,13 @@
     "hoek": "^4.2.1",
     "react": "^16.4.1",
     "react-dom": "^16.4.1"
-  }
+  },
+  "browserslist": [
```

Comment:
So in relation to declaring the browsers, how would it be declared? In terms of syntax e.g. chrome-65

Reply:
This link may help to explain this better:
https://github.com/browserslist/browserslist

It's unlikely you would want to specify a specific version of Chrome or Firefox, as the versions are constantly being updated. Within the browserslist, the version of Chrome has been set to the last 2 versions, so this will cover the most recent versions only. You can of course change this if you want to, but you shouldn't need to specify every browser version.